website/client/components/guilds/myGuilds.vue

```diff
@@ -0,0 +1,125 @@
+<template lang="pug">
+.row
+  sidebar(v-on:search="updateSearch", v-on:filter="updateFilters")
+
+  .col-10.no-guilds(v-if='filteredGuilds.length === 0')
+    .no-guilds-wrapper
+      img(src='~assets/guilds/grey-badge.svg')
+      h2 {{$t('noGuildsTitle')}}
+      p {{$t('noGuildsParagraph1')}}
+      p {{$t('noGuildsParagraph2')}}
+      span(v-if='loading') {{ $t('loading') }}
+
+  .col-10(v-if='filteredGuilds.length > 0')
+    .row
+      .col-md-12
+        h1.page-header.float-left(v-once) {{ $t('myGuilds') }}
+        b-form-select.float-right.sort-select(v-model='sort', :options='sortOptions')
+    .row
+      .col-md-12
+        public-guild-item(v-for="guild in filteredGuilds", :key='guild._id', :guild="guild", :display-gem-bank='true')
+        mugen-scroll(
+          :handler="fetchGuilds",
+          :should-handle="loading === false && hasLoadedAllGuilds === false",
+          :handle-on-mount="false",
+          v-show="hasLoadedAllGuilds === false",
+        )
+          span {{ $t('loading') }}
+</template>
+
+<style lang="scss" scoped>
+.sort-select {
+  margin: 2em;
+}
+
+.no-guilds {
+  text-align: center;
+  color: #878190;
+  margin-top: 15em;
+
+  p {
+    font-size: 14px;
+    line-height: 1.43;
+  }
+
+  .no-guilds-wrapper {
+    width: 400px;
+    margin: 0 auto;
+  }
+}
+</style>
+
+<script>
+import { mapState } from 'client/libs/store';
+import groupUtilities from 'client/mixins/groupsUtilities';
+
+import MugenScroll from 'vue-mugen-scroll';
+import bFormSelect from 'bootstrap-vue/lib/components/form-select';
+
+import PublicGuildItem from './publicGuildItem';
+import Sidebar from './sidebar';
+
+export default {
+  mixins: [groupUtilities],
+  components: { PublicGuildItem, MugenScroll, Sidebar, bFormSelect },
+  data () {
+    return {
+      loading: false,
```

Comment:
It is pretty abstracted, and actually, we talked about removing paging from my guilds - so I think I can just remove most of this, right?